src/content/learn/thinking-in-react.md

```diff
@@ -391,7 +387,7 @@ function SearchBar({ filterText, inStockOnly }) {
     <form>
       <input 
         type="text" 
-        value={filterText} 
+        thamani={filterText} 
```

Comment:
Just a comment, could we maintain the original language syntax i.e. in this case, the language syntax is in JavaScript